tests/cases/compiler/constEnumErrors.ts

```diff
@@ -9,10 +9,10 @@ module E {
 const enum E1 {
     // illegal case
     // forward reference to the element of the same enum
-    X = Y, 
+    X = Y, // error

```

Comment:
In the future, if you're going to amend tests, please do it in two separate commits (1 for change of tests, 1 for baselines) so that the diff is more readable.